input/README.md

```diff
@@ -18,6 +18,31 @@ C	2.5		True	True
 D	1.9	True	True	True
 ```
 
+##### OmicsIntegrator1: Dummy Nodes
+There are 4 dummy mode possibilities:
+ 1. terminals -> connect the dummy node to all nodes that have been assigned prizes 
+ 2. all ->  connect the dummy node to all nodes in the interactome i.e. full set of nodes in graph
+ 3. others -> connect the dummy node to all nodes that are not terminal nodes i.e. nodes w/o prizes
+ 4. file -> custom nodes - connect the dummy node to a specific list of nodes provided in a file
+To support the `file` dummy node logic as part of OmicsIntegrator1, you can either add a seperate `dummy.txt` file (and add this to the `node_files` argument in `config.yaml `) or add a `dummy` column node attribute to a file that contains `NODEID`, `prize`, `source`, etc. 
+
+If adding a seperate `dummy.txt` file:
+Make a file with the name `dummy.txt` and list the dummy nodes, each seperated by a new line. Example:
+```
+A
+B
+C
+```
+
+If adding the `dummy` column node attribute, then add the dummy column and specify boolean values for the `dummy` attribute:
+```
+NODEID	prize	sources	targets	dummy
+A	1.0		True	True    True
+B	3.3	True	True	
+C	2.5		True	True
+D	1.9	True	True	True
+```
```

Comment:
We don't need to copies of this node table. We can replace the copy above with this new one that has the dummy column.